react/react.d.ts

```diff
@@ -278,6 +278,8 @@ declare namespace __React {
         preventDefault(): void;
         stopPropagation(): void;
         persist(): void;
+        isDefaultPrevented(): boolean;
+        isPropagationStopped(): boolean;
```

Comment:
small nit: can you use the same order as in the documentation? ie. `isDefaultPrevented` after `preventDefault`, and `isPropagationStopped` after `stopPropagation`. thanks!